ports/lpc/chips/lpc_chip_43xx/inc/iap_18xx_43xx.h

```diff
@@ -79,7 +79,7 @@ extern "C" {
 #define IAP_CRP_ENABLED             19	/*!< Code read protection enabled */
 
 /* IAP_ENTRY API function type */
-typedef void (*IAP_ENTRY_T)(unsigned int[5], unsigned int[4]);
+typedef void (*IAP_ENTRY_T)(uint32_t[5], uint32_t[4]);
```

Comment:
esto lo agarré de lpcopen. no lo toqué. porque el cambio?

Reply:
El compiador tiene -Werror activado. Hay que sacarle todo lo que de warns.
Esto particularmente hacia problemas de type convertion (rarisimo que no se de cuenta que es equivalente)

Reply:
Perfecto. Como dije en el comentario, queremos poner esto en un commit aparte y explicar porque. Es importante para cuando hagamos un drop de una version nueva de lpcopen, y vamos a querer re-aplicar estos fixes. Yo lo puedo hacer si querés. Puede ser un commit que diga "Fix all the crap in LPCOpen so the compiler stops complaining".

Reply:
Dale, ahora estoy en el trabajo y switcheo de una cosa a otra :-(